components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2791,6 +2813,31 @@ where
         ))
     }
 
+    fn exec_flashback_apply(
+        &self,
+        ctx: &mut ApplyContext<EK>,
+        _req: &AdminRequest,
+        state: FlashbackState,
+        result: ExecResult<EK::Snapshot>,
+    ) -> Result<(AdminResponse, ApplyResult<EK::Snapshot>)> {
+        let region = self.region.clone();
+        write_peer_state(
+            ctx.kv_wb_mut(),
+            &region,
+            PeerState::Normal,
+            None,
```

Comment:
Will these values overwrite the original states? Shouldn't we try to read the old values first and then just write to the flashback state?